app/components/AuctionBidBox.js

```diff
@@ -109,177 +76,122 @@ export default class AuctionBidBox extends Component {
 
   @computed
   get downBid() {
-    return this.newBid.minus(this.props.bidIncrement);
+    const { bidIncrement } = this.props.auctionStore.auction;
```

Comment:
Maybe all of these should be in the `auctionStore`

Reply:
In that case, we also wouldn't need to inject `this.props.store` probably.

Reply:
Eh I like keeping the `newBid` as a `AuctionBidBox` variable, but I'll refactor more stuff into `AuctionStore`